Back-End/controllers/instructors_controllers/courses_controllers.js

```diff
@@ -1,5 +1,16 @@
 const db = require('../../db');
 
+const registeredCoursesInstructor = (req, res) => {
+    const query = `SELECT * FROM courses c JOIN registration r ON c.id = r.course_id AND instructor_id = ?`;
+    const data = req.params.id;
+    db.query(query, data, (err, result) => {
+        if (err) throw err
+        res.json(result)
+    })
+};
+
+
+
```

Comment:
These 2 lines are extra, make sure to remove them

Reply:
Noted.